examples/playlist/schemas/kind.json

```diff
@@ -0,0 +1,36 @@
+{
+  "group": "playlists.ext.grafana.com",
+  "kind": "Playlist",
+  "description": "A set of dashboards that will be displayed in a loop (dummy for testing)",
+  "maturity": "merged",
+  "machineName": {
```

Comment:
I do understand that this is heavily inspired from K8 but for me `machineName` feels weird to use in this context.

Reply:
ya -- it is a weird one, but needed for k8s 😬   Note that "composable" kinds do not need any of the machine stuff